udata/core/badges/api.py

```diff
@@ -3,8 +3,43 @@
 
 from udata.api import api, fields
 
+from .forms import badge_form
+
+
 badge_fields = api.model('Badge', {
     'kind': fields.String(description=('Kind of badge (certified, etc), '
                                        'specific to each model'),
                           required=True),
 })
+
+
+def add_badge_api(obj):
```

Comment:
I'm not fond of that naming. Proposition with pseudo-code:

* `badges/api.py`: `add` and `remove`
* `badges/__init__.py`: `import add, remove`
* `dataset/anywhere.py`: `from core import badges` then in code `badges.add()`

Thoughts?